### PR TITLE
PR for Manjaro

### PR DESCRIPTION
**Pull request by dreaddr**
As requested, to show Manjaro on Quick Setup on #370.

### CODE REVIEW

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]: This can be removed, right? As you said, Flatpak is installed by default on Manjaro.

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]: I would rewrite this to say for example: `Flatpak is installed by default on Manjaro 20 or higher. To enable its support, navigate to the <strong>Software manager</strong>(Add/Remove Programs) ...`

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml
Comment (dreaddr) [reply data/distro.yml]: Sure

File: data/distro.yml
Comment (dreaddr) [reply data/distro.yml]: Yeah that makes sense, I'll make the changes

File: data/distro.yml
Comment (dreaddr) [reply data/distro.yml]: Although, advanced users of Manjaro might just want terminal commands, but it should be fine.

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]: This note is not needed. We do not mention it if the distro has Flathub enabled out-of-box.

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]: This is not needed as well. It is already mentioned in the "Restart" section.

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml
Comment (dreaddr) [reply data/distro.yml]: Okay, will fix these, any other issues you noticed before I do?

Comment (AsciiWolf) [reply data/distro.yml]: It looks fine otherwise.

**Comment by AsciiWolf** [comment]
Oh, one more thing: You probably forgot to include manjaro.svg in this PR. :-)

**Comment by dreaddr** [comment]
Should be good now

**Comment by AsciiWolf** [comment]
Thanks! Probably LGTM now, but I will keep this opened for another day or two before merging. Maybe someone else (@barthalion ?) will also review it.

**Comment by dreaddr** [comment]
Alright then, hope it merges ASAP

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
        <p>To enable its support, navigate to the <strong>Software Manager</strong> (Add/Remove Programs)</p>
```

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
        <p>Navigate to the "Flatpak" tab and slide the toggle to Enable Flatpak support (it is also possible to enable checking for updates, which is recommended).</p>
```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
Two more small things. I will merge this PR after they are resolved.

**Comment by AsciiWolf** [comment]
Build failed because of an unrelated issue: https://github.com/flatpak/flatpak.github.io/issues/468

**Comment by dreaddr** [comment]
I assume I don't need to do anything more?

**Comment by AsciiWolf** [comment]
Yeah, this needs to be fixed on our side. :-)

**Comment by dreaddr** [comment]
Just checking up, is this live now?

**Comment by AsciiWolf** [comment]
Nope, it seems to be still waiting on #468. Sorry.

**Comment by dreaddr** [comment]
Oh I see, just checking. Would be nice to hear when it goes live though.

**Pull request merged**
